lib/src/base-styles/_nx-tile.scss

```diff
@@ -157,8 +157,21 @@ $nx-tile-background-color: $nx-white;
 
   border-top: 1px solid #e9e9e9;
   margin-top: $nx-spacing-md;
```

Comment:
Looking at the tile mocks in zeplin, this should be 24

Reply:
> Looking at the tile mocks in zeplin, this should be 24

https://github.com/sonatype/sonatype-react-shared-components/pull/124/commits/85709f44f4f557bcdf54b5fdc99486971fc1441c